charts/vault-monitoring/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: vault-monitoring
 description: monitor your vault server from within Kubernetes' prometheus
 type: application
-version: 0.3.1
+version: 0.3.2
```

Introducing a new alert constitutes a new feature, so I'd say we should increase the minor version.
```suggestion
version: 0.4.0
```